src/components/templates/ArtPiece/ArtPiece.scss

```diff
@@ -75,6 +86,7 @@ $chat-wrapper-height: 260px;
       width: unset;
       flex-grow: 0;
       flex-shrink: 0;
+      border-radius: $spacing--xl;
```

Comment:
Not sure if rounded corners are / will be OK with the content displayed inside

Reply:
looks alright atm